### DISCUSSION TITLE
Overall logging framework expansion

### DISCUSSION THREAD

**Discussion by colaso96** [microsoft/autogen#2730] [General]
Hello, I have implemented an agent system built with autogen to write a variation of SQL queries.  I am looking to expand the user base and improve the logging of each interaction or action the agent(s) take.  So far the only documentation I could find is: https://microsoft.github.io/autogen/docs/notebooks/agentchat_logging/.  This is a good start but ideally I would like to log all the function calls that the system makes as well as the parameters and arguments.  Is there a plan to implement something like this?

**Comment by ekzhu** [comment]
Yes this is related to #2423, it is part of our roadmap #7 . You can check out the recent activities in this area by using the `logging` lable, e.g., #2596
